src/core/utils.js

```diff
@@ -277,6 +278,40 @@ export function joinAnd(array = [], mapper = item => item, lang = docLang) {
     }
   }
 }
+// STRING HELPERS
```

Comment:
Not exactly a string helper as the function receives `HTMLElement`s. Could you move this function into `DOM helpers` area?